pyblish/lib.py

```diff
@@ -53,10 +53,32 @@ def emit(self, record):
             self.records.append(record)
 
 
-def extract_traceback(exception):
-    """Inject current traceback and store in exception"""
+def extract_traceback(exception, plugin):
```

Comment:
Sorry to keep picking on this, but something only become apparent when you're just about to merge. xD

Could we make this function take a `fname=None` argument instead? Having it take `plugin` implies some dependency on Pyblish's Plugin class, and it's actually quite general and has to do with overriding the `fname` found in the `exception` argument. That would make it squeaky clean.

If you're done with this, I understand and could take a look at it myself in the coming days.

Reply:
No worries, I'll add that later today

Reply:
It's changed